tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+skipsdist = True
+envlist = flake8
+
+[testenv]
+deps =
+    -rrequirements.txt
+    -rtest-requirements.txt
+commands = pytest {posargs}
+
+[testenv:flake8]
+deps = flake8
+commands = flake8 pureport
+
+[flake8]
+max-line-length = 140
```

Comment:
140 characters seems rather large. I'd prefer a limit of 100, but I think 120 is okay too.

Reply:
I think I have some punchcards I inherited from my mom laying around if you'd like to go to 80. 

120 sounds good 😄